service/local-dev/skaffold.yaml.template

```diff
@@ -4,7 +4,7 @@ kind: Config
 build:
   artifacts:
   - image: gcr.io/terra-kernel-k8s/terra-workspace-manager
-    context: ../
+    context: ../../
```

Comment:
is there a way to refer to the root dir directly so we don't have to rely on upward references?

Reply:
I wish there were. I tried to figure one out for the application property files when I moved `render` -> `../config`, but didn't find a way.

We could use an envvar and try to set it properly in all of the places we make these references, but that might just make things harder to figure out.